site/docs/concepts/beam-search.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 3
+---
+
+# Beam search
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident.
```

Comment:
Remove the text until it's rewritten.

Reply:
Are you suggesting to remove all text placeholders (lorem ipsum ...) in all not ready pages and keep only headers for initial structure? Or remove all not ready pages at all and do not display them?

Reply:
Remove all text placeholders (lorem ipsum ...) in all not ready pages and keep only headers for initial structure?

We'd need these pages anyway later.

Reply:
Text removed for all incomplete pages